### PR TITLE
SRE-2510: add license as expected by OpenSSF Scorecard

### PR DESCRIPTION
**Pull request by grom72**
Inherit BSD-2-Clause-Patent License
from the main - daos-stack/daos project.

### CODE REVIEW

**Review by janekmi — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (c) 2019-2024 Intel Corporation
```
Comment (janekmi) [inline Makefile:2]:
```suggestion
# Copyright (c) 2019-2024 Intel Corporation

```

**Review by janekmi — CHANGES_REQUESTED** [review]

**Review by janekmi — APPROVED** [review]

**Review by osalyk — APPROVED** [review]

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
This failed to build.  You probably need to update the `packaging/` subdir from https://github.com/daos-stack/packaging/ by [cloning and] checking out `master` from that somewhere and then in the directory for this PR run `$path/to/packaging/master/install`.

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
The CentOS 7 build seems to be failing still.  But the good news is that we can simply just stop building this on CentOS 7.

**Comment by grom72** [comment]
> The CentOS 7 build seems to be failing still. But the good news is that we can simply just stop building this on CentOS 7.

Done

**Comment by grom72** [comment]
Hi @daos-stack/release-engineering,
all tests passed. 
PR is ready for landing.

**Pull request merged**